packages/yourloops/components/patient/add-patient-dialog.tsx

```diff
@@ -68,19 +68,13 @@ export const AddPatientDialog: FunctionComponent<AddDialogProps> = ({ onClose, o
   const isValidEmail = (mail = email): boolean => mail.length > 0 && REGEX_EMAIL.test(mail)
 
   const handleClickAdd = async (): Promise<void> => {
-    if (isValidEmail() && selectedTeam.id.length > 0) {
+    if (isValidEmail()) {
       await addPatient()
     } else {
       setErrorMessage(t('invalid-email'))
     }
   }
 
-  const handleVerifyEmail = (): void => {
-    if (email.length > 0 && !isValidEmail()) {
-      setErrorMessage(t('invalid-email'))
-    }
-  }
-
   const handleChangeEmail = (event: React.ChangeEvent<HTMLInputElement>): void => {
     const inputEmail = event.target.value.trim()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏I suggest to add here the `invalid-email` error message, we have lost it after removing `handleVerifyEmail` method, and the line 74 is never called !